src/test/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmControllerTest.java

```diff
@@ -691,15 +692,16 @@ private HttpHeaders getHttpHeaders() {
 
     @Test
     void generateHashCode_HappyPath() {
+        Optional<StoredDocumentHalResource> documentMetadata = setDocumentMetaData();
 
-        doReturn(setDocumentMetaData()).when(documentManagementService).getDocumentMetadata(getUuid());
+        doReturn(documentMetadata).when(documentManagementService).getDocumentMetadata(getUuid());
```

Comment:
If we change `setDocumentMetaData` to return `StoredDocumentHalResource`, then here we can do

`doReturn(Optional.of(documentMetadata)).when(documentManagementService).getDocumentMetadata(getUuid());`